src/components/Comments/WriteComment.js

```diff
@@ -15,10 +15,11 @@ import ToggleSwitch from '../ToggleSwitch';
 import AnonUserInfoRepo from '../../service/repo/anonUserInfoRepo';
 import MemoSendComment from '../../assets/icon/IconSendComment';
 import StringConstant from '../../utils/string/StringConstant';
+import {ToggleSwitchAnon} from '../../hooks/useToggleColors';
 import {Context} from '../../context';
-import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` has been updated correctly from the old `colors`. This change is consistent with the PR summary and improves the code maintainability by using a more descriptive name.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->